tracing/opentracing/src/main/java/io/helidon/tracing/opentracing/OpenTracingSpan.java

```diff
@@ -66,7 +64,7 @@ public void status(Status status) {
 
     @Override
     public SpanContext context() {
-        return context;
+        return Span.current().orElseThrow().context();
```

Comment:
Why not return the wrapped delegate.context() here? (i.e. the line removed in above constructor). Aren't you unnecessarily risking a throw here?

Reply:
This is outdated.